docs/getting-started/core-concepts/teams.md

```diff
@@ -5,24 +5,24 @@ weight: 2
 
 ## What are Teams?
 
-`Teams` in a Greenhouse `Organization` are used to group users by `group` claims on the token provided by the upsream identity provider (IdP).
+Teams in a Greenhouse Organization are used to group users by `group` claims on the token provided by the upstream identity provider (IdP).
 
-This can be used for i.a.
+This can be used, for example, for
 
 - organizational management
 - access and permission management
 - identifying [`Ownership` of resources](./../operations/ownership.md)
 
-The Greenhouse UI is showing the members of a `Team`.
+The Greenhouse UI is showing the members of a Team.
 
 ## Team RBAC
 
-`TeamRoles` and `TeamRoleBindings`  provide a mechanism to control the permissions of Teams to onboarded Clusters of an Organization.
+TeamRoles and TeamRoleBindings  provide a mechanism to control the permissions of Teams to onboarded Clusters of an Organization.
 
-`Team` role-based access control (RBAC) wraps the concept of Kubernetes RBAC in `TeamRoles` and `TeamRoleBindings` . `TeamRoles` are used to define a set of RBAC permissions. These permissions can be granted to Teams with `TeamRoleBindings` . A `TeamRoleBinding` refers to a Team, a `TeamRole` , Cluster(s) and optional Namespaces. Depending on the latter, Greenhouse will create the appropriate `rbacv1` resources on the targeted cluster(s) in either Cluster or Namespace scope.
+Team role-based access control (RBAC) wraps the concept of Kubernetes RBAC in TeamRoles and TeamRoleBindings . TeamRoles are used to define a set of RBAC permissions. These permissions can be granted to Teams with TeamRoleBindings . A TeamRoleBinding refers to a Team, a TeamRole , Cluster(s) and optional Namespaces. Depending on the latter, Greenhouse will create the appropriate `rbacv1` resources on the targeted cluster(s) in either Cluster or Namespace scope.
 More information about how this can be configured is mentioned in this [user guide](../../user-guides/team/rbac.md).
 
-Example of a `TeamRoleBinding` for a `observability-admin` which grants the `cluster-admin` role on the `observability` cluster in the `logs` and `metrics` namespaces. The `TeamRoleBinding` contains a list of namespaces and a label selector to select the cluster(s) to target. If no Namespaces are provided, then Greenhouse will create a `ClusterRoleBinding` instead of a `RoleBinding`.
+Example of a TeamRoleBinding for a `observability-admin` which grants the `cluster-admin` role on the `observability` cluster in the `logs` and `metrics` namespaces. The TeamRoleBinding contains a list of namespaces and a label selector to select the cluster(s) to target. If no Namespaces are provided, then Greenhouse will create a `ClusterRoleBinding` instead of a `RoleBinding`.
```

Comment:
```suggestion
Example of a TeamRoleBinding for a `observability-admin` which grants the `cluster-admin` role on the `observability` cluster in the `logs` and `metrics` namespaces. The TeamRoleBinding contains a list of namespaces and a label selector to select the cluster(s) to target. If no Namespaces are provided, then Greenhouse will create a ClusterRoleBinding instead of a RoleBinding.
```